tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -3948,4 +3948,151 @@ TestCarePartnerParticipantInfoEditOrAddEmergencyRoomAdmissionsInfo:
       Id: $PAGE.care_platform_edit_participant_hospitalization.delete_emergency_room_admission_dialog_title$
     - Type: wait_for
       Strategy: xpath
-      Id: $PAGE.care_platform_notifications.emergency_room_admission_deleted_notification$
\ No newline at end of file
+      Id: $PAGE.care_platform_notifications.emergency_room_admission_deleted_notification$
+
+#CXTB-111 Participant Profile - Review, Edit or Add Vendor Info
+TestReviewEditAddVendorInfo:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+  Actions:
+  # Calling Login case.
+    - Type: case
+      Value: CarePartnerLogin
+  # looking for a senior profile and click on it
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_participant$
+    - Type: click 
+      Strategy: xpath
+      Id: $PAGE.care_platform_home.first_call_of_queue$
+  # Click on "records" button 
+    - Type: click
+      Strategy: xpath 
+      Id: $PAGE.care_platform_participant_details.records_button$
+  # click on "Demographic" and "VENDORS" tabs
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_records.demographics_tab$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_records.demographics_vendor_subtab$
+  # variables to use for Add Vendor
+      Vars:
+        ADDRESS: 1000 Street Name, Somewhere, GA 30024
+        WEBSITE: www.airportridesrus.com
+        CATEGORY: careCoordination
+        PHONE: 4365565897
+    - Type: generate_unique_name
+      Name: 'Airport Rides "R" Us'
+      ResultVar: NAME
+  # Steps to add a vendor
+    # clicking on "Add Vendor" button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.add_vendor_button$
+    # waiting for modal
+    - Type: wait_for
+      Strategy: id
+      Id: $PAGE.care_platform_edit_participants_vendor.vendor_modal$
+    # closing "Add Vendor" modal
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_edit_participants_vendor.cancel_vendor_modal_button$
```

Comment:
it would be good to match the **expected results** from XRay, on this case it is expecting that the modal closes after the Cancel button is clicked. If we are not verifying that it is not visible anymore then we are just clicking around.

Reply:
This is still not solved